packages/material-ui-lab/src/TimelineItem/TimelineItem.js

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { capitalize, isMuiElement } from '@material-ui/core/utils';
+import { withStyles } from '@material-ui/core/styles';
+import TimelineContext from '../Timeline/TimelineContext';
+
+export const styles = () => ({
+  /* Styles applied to the root element. */
+  root: {
+    listStyle: 'none',
+    display: 'flex',
+    position: 'relative',
+    minHeight: 70,
+  },
+  /* Styles applied to the root element if `align="left"`. */
+  alignLeft: {},
+  /* Styles applied to the root element if `align="right"`. */
+  alignRight: {
+    flexDirection: 'row-reverse',
+  },
+  /* Styles applied to the root element if `align="alternate"`. */
+  alignAlternate: {
+    '&:nth-child(even)': {
+      flexDirection: 'row-reverse',
+      '& .MuiTimelineItemContent-root': {
+        textAlign: 'right',
+      },
+      '& .MuiTimelineItemOppositeContent-root': {
+        textAlign: 'left',
+      },
+    },
+  },
+  /* Styles applied to the root element if no there isn't TimelineItemOppositeContent provided. */
+  missingOppositeContent: {
+    '&:before': {
+      content: "''",
```

Comment:
Quotes need to be reversed for IE 11 support

```suggestion
      content: '""',
```